src/main/java/io/github/christophermanahan/carnitas/Router.java

```diff
@@ -15,20 +15,25 @@
     }
 
     Router get(String uri, Function<HTTPRequest, HTTPResponse> handler) {
-        map.put(new Route(HTTPRequest.Method.GET, uri), handler);
+        mapRoute(HTTPRequest.Method.GET, uri, handler);
         return this;
     }
 
     Router head(String uri, Function<HTTPRequest, HTTPResponse> handler) {
-        map.put(new Route(HTTPRequest.Method.HEAD, uri), handler);
+        mapRoute(HTTPRequest.Method.HEAD, uri, handler);
         return this;
     }
 
     Router post(String uri, Function<HTTPRequest, HTTPResponse> handler) {
-        map.put(new Route(HTTPRequest.Method.POST, uri), handler);
+        mapRoute(HTTPRequest.Method.POST, uri, handler);
         return this;
     }
 
+    private void mapRoute(HTTPRequest.Method method, String uri, Function<HTTPRequest, HTTPResponse> handler) {
```

Comment:
@damonkelley I originally wanted to use route, but it would be confusing as well because it would be used as a verb here, but we also have the Route data structure.

Reply:
I am considering `add`